### PR TITLE
Rover: Added PID logging of throttle

### PR DESCRIPTION
**Pull request by gmorph**

### CODE REVIEW

File: APMrover2/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -363,15 +363,29 @@ void Rover::send_rangefinder(mavlink_channel_t chan)
...
-                                    pid_info.D);
+                                    pid_info->FF,
+                                    pid_info->P,
+                                    pid_info->I,
+                                    pid_info->D);
+        if (!HAVE_PAYLOAD_SPACE(chan, PID_TUNING)) {
+            return;
+        }
+    }
+    if (g.gcs_pid_mask & 2) {
```
Comment (magicrub) [inline APMrover2/GCS_Mavlink.cpp]: Please use a bitmask define instead of a magic number.

Comment (gmorph) [inline APMrover2/GCS_Mavlink.cpp]: I will as soon as you change plane over to do the same :-D

**Comment by magicrub** [comment]
Them fight'n words!  Hehe

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**